README.md

```diff
@@ -45,7 +45,7 @@ as we can generate valid dockerfile for it):
 
 ```bash
 ./makejdk-any-platform.sh --docker --clean-docker-build jdk8u
-./makejdk-any-platform.sh --docker --clean-docker-build --build-variant openj9 jdk11u
+./makejdk-any-platform.sh --podman --clean-docker-build --build-variant openj9 jdk11u
```

Comment:
Any reason not to make this `-D` now?
```suggestion
./makejdk-any-platform.sh -D --clean-docker-build --build-variant openj9 jdk11u
```

Reply:
I had left `-D` to autodetect. Id there is podman, it will be used. If not, docekr will be used.  Reason for this fallback is that if you have podman, you have also docker aliases. But if you have docker, you do no thave podman aliases.

Reply:
Yep understood - I just wasn't sure why this example was explicitly switched to `--podman` instead of showing the "autodetect" version

Reply:
I jsut wanted to keep all three there. If they should be reuced, jsut say.

Reply:
I have a slight preference fur using the generic `-D`, although I quite like the multiple examples too, so no real preference with your explaination :-)